Makefile

```diff
@@ -62,7 +62,8 @@ submodules:
 rapidwright: submodules
 	cd third_party/RapidWright && ./gradlew compileJava
 	cd third_party/RapidWright/interchange/ && make
-	$(IN_ENV) python scripts/rapidwright_get_data_files.py
+	$(IN_ENV) cd third_party/RapidWright && ./gradlew compileJava ; export PATH=`pwd`/bin:$$PATH ; \
```

Comment:
I don't think we need the `&& ./gradlew compileJava` part as we have it already up on line 63.